back/Dockerfile

```diff
@@ -5,7 +5,7 @@ MAINTAINER Jean-Louis Rigau <jlrigau@xebia.fr>
 RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list
 
 RUN apt-get update
-RUN apt-get install bcrypt node-gyp -y
+RUN apt-get install bcrypt node-gyp -y --force-yes
```

Comment:
I guess it's not needed? I want to know if there is an error ;)

Reply:
There is an error when you don't put --force-yes :(

Reply:
Is it working even with the error? I.e. app is running?